DataAccess/DataAccess.Postgre/DataBaseOptions.hpp

```diff
@@ -4,7 +4,7 @@
 namespace DBOptions
 {
     constexpr std::string_view real = 
-        "dbname=juniorgram user=postgres hostaddr=127.0.0.1 port=5432";
```

Comment:
Was there any change to our database naming? Why did it had to happen? Currently that DB is empty:
<img width="476" alt="image" src="https://user-images.githubusercontent.com/13378692/127662554-f42234d8-ed95-4243-87cf-d4dc0cfff235.png">